src/pages/ExamsPage/components/AllowanceListActions.test.jsx

```diff
@@ -18,6 +11,19 @@ const mockAllowance = {
   extraTimeMins: 45,
 };
 
+jest.mock('../hooks', () => ({
+  useDeleteAllowance: jest.fn(),
+  useEditAllowance: jest.fn(),
+  useAllowancesData: jest.fn().mockReturnValue({
```

Comment:
Question: is there a better way to mock this? When I try this way, I get undefined when I try to filter by allowance id.

Reply:
I think this needs to return an array. That said, see my comment above because it doesn't seem to like we actually need this data.